src/odemis/acq/calibration.py

```diff
@@ -237,7 +237,7 @@ def get_spectrum_efficiency(das):
     return ret
 
 
-def spectrum_efficiency_and_bg_correction(data, bckg=None, coef=None):
+def apply_corrections(data, bckg=None, coef=None):
```

Comment:
Well, the name should indicate it processes spectra. => `apply_spectrum_corrections()`?

Reply:
My intention was to keep it generic in case we support time correlator corrections, but as we do not for now I changed it  accordingly.